src/hooks/useActiveAccount.tsx

```diff
@@ -82,6 +83,13 @@ export const ActiveAccountContextProvider = ({
     };
   }, [cache, setPreferredId]);
 
+  /**
+   * This check is temporary. It will be refactored out in a subsequent PR.
+   */
+  if (accountsResult.status === 'success' && !selectedAccount) {
+    return <InviteRequiredScreen />;
+  }
```

Comment:
This approach is not as hacky as I thought it was when I wrote this comment. But, regardless: when the shift to "static account definition" happens, this code will be factored out.